Musicplayer.css

```diff
@@ -1,9 +1,12 @@
+/* Import  font from google fonts*/
```

Comment:
21/9: I import font from google fonts. The fonts I choice is Source sans pro and Open sans